go.mod

```diff
@@ -20,6 +21,7 @@ require (
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/jwalterweatherman v1.1.0 // indirect
 	github.com/spf13/pflag v1.0.5
+	github.com/vincent-petithory/dataurl v0.0.0-20191104211930-d1553a71de50 // indirect
```

Comment:
This seems odd - I think if you delete this line and run `go mod tidy` it should disappear.

Reply:
This is pulled via https://github.com/coreos/ignition/blob/spec2x/config/v2_3/types/url.go#L20.